pkg/azurelustreplugin/fix-lnet.sh

```diff
@@ -1,12 +1,28 @@
 #!/bin/bash
 
-for run_id in 0 2 4; do
-  sleep $run_id
-  (
-    flock -w 60 -e ${FD}    
-    if sudo lnetctl net show --net tcp | grep "status: down"; then
-      /usr/sbin/lnetctl net del --net tcp
-      /usr/sbin/lnetctl net add --net tcp --if {default_interface}
-    fi
-  ) {FD}< /etc/lustre/.lock
+logger "PID $$: Start fix-lnet"
```

Comment:
I think it's better to specify /usr/bin/logger because this script won't be triggered in a shell. So that, the $PATH may be incorrect.

Reply:
added